src/temp_sensor.rs

```diff
@@ -0,0 +1,38 @@
+#[cfg(esp32c3)]
+use esp_idf_sys::{
+    soc_periph_temperature_sensor_clk_src_t,
+    soc_periph_temperature_sensor_clk_src_t_TEMPERATURE_SENSOR_CLK_SRC_DEFAULT,
+};
+#[cfg(esp32c3)]
+pub type TemperatureSensorConfig = config::Config;
+
+/// Temperature Sensor configuration
+#[cfg(esp32c3)]
+pub mod config {
+    #[derive(Copy, Clone)]
```

Comment:
you need to import here with either `use super::*` or the better way the module or directly move the above use inside the module so you dont need extra scopes

Reply:
I see. That was the missing information. Thanks!

Reply:
Resolved by https://github.com/esp-rs/esp-idf-hal/pull/337/commits/01ce494260da154db5794c89a94ec77bb9bdef14.